components/tests.py

```diff
@@ -37,27 +37,27 @@
                         "implemented-requirements": [
                             {
                                 "uuid": "6698d762-5cdc-452e-9f9e-3074df5292c6",
-                                "control-id": "ac-2.1",
+                                "control-id": "ac-1",
                                 "description": "This component statisfies a.",
                             },
                             {
                                 "uuid": "73dd3c2e-54eb-43c6-a488-dfb7c79d9413",
-                                "control-id": "ac-2.2",
+                                "control-id": "ac-2",
                                 "description": "This component statisfies b.",
                             },
                             {
                                 "uuid": "73dd3c2e-54eb-43c6-a488-dfb7c79d9413",
-                                "control-id": "ac-2.3",
+                                "control-id": "at-1",
```

Comment:
Remind me why the control id needed to change? I think you mentioned this, but I forget.

Reply:
For the catalog and components controls to be aligned. Nothing checked this before